src/api/comments.ts

```diff
@@ -0,0 +1,20 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: Comment['postId']) => {
```

Comment:
Consider adding error handling to the `getComments` function to manage potential API request failures gracefully.